store/tikv/snapshot.go

```diff
@@ -355,7 +350,11 @@ func (s *tikvSnapshot) get(bo *Backoffer, k kv.Key) ([]byte, error) {
 		}
 	}
 	s.mu.RUnlock()
-
+	if span := opentracing.SpanFromContext(ctx); span != nil && span.Tracer() != nil {
```

Comment:
What is this change for?

Reply:
In order to prevent the tikvSnapshot.get()` repeated too many times

Reply:
Each `tikvSnapshot.Get` calls `tikvSnapshot.get` once, would it help?